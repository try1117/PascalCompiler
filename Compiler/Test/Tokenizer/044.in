1.2345.e