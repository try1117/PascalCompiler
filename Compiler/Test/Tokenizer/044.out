1  | 1  | CONST_DOUBLE             | 1.2345                   | 1.234500000000000
1  | 7  | SEP_DOT                  | .                        | 
1  | 8  | IDENTIFIER               | e                        | 
